WebAPI/WebAPI.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Serilog.Sinks.Async" Version="1.4.0" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
     <PackageReference Include="Serilog.Sinks.File" Version="4.1.0" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.6.0-pre4" />
```

Comment:
It shouldn't be there because Xamarin is dependency **only** for mobile application, not API.